src/solver/optim/ortoolsImpl/include/antares/solver/optim/ortoolsImpl/linearProblem.h

```diff
@@ -66,7 +65,7 @@ class OrtoolsLinearProblem final: public Api::ILinearProblem
     OrtoolsMipSolution* solve(bool verboseSolver) override;
 
 private:
-    std::unique_ptr<operations_research::MPSolver> mpSolver_;
+    std::shared_ptr<operations_research::MPSolver> mpSolver_;
```

Comment:
I would rather use a unique pointer to insure this object will only live with LinearProblem
You can still pass the raw pointer to MipSolution